Client/src/RTypeGameContext.cpp

```diff
@@ -23,15 +30,50 @@ void RTypeGameContext::Setup(std::string const &partitionFile) {
         throw new std::runtime_error("Invalid partition file");
 
     _pool->LoadPartition(data);
+
+    _eventListener->Subscribe<void, ReceivedNetworkPayloadMessage>(ReceivedNetworkPayloadMessage::EventType, [&](void *sender, ReceivedNetworkPayloadMessage *message) {
+        auto packet = RType::Packer(RType::READ, message->getPayload()->Payload);
+        EntityPacker entityPacker(packet, _pool->getFactory());
+
+        if (_pool->Exist(entityPacker.getEntityId()) && !_pool->isPlayer(entityPacker.getEntityId()))
+            return; //Drop the packet
+
+        entityPacker.UnpackEntity(_timer, _pool->getEventManager());
+        _mailbox.enqueue(entityPacker);
+    });
+
+    _eventListener->Subscribe<Entity, UserInputMessage>(UserInputMessage::EventType, [&](Entity *, UserInputMessage *message) {
+        RType::Packer packer(RType::WRITE);
+        message->Serialize(packer);
+        _eventManager->Emit(SendNetworkPayloadMessage::EventType, new SendNetworkPayloadMessage(packer, "127.0.0.1"), this);
```

Comment:
Along with the previous case, we should start using a variable instead of `127.0.0.1`